lib/components/Upload.js

```diff
@@ -293,6 +343,28 @@ export default class Upload extends Component {
     );
   }
 
+  renderMedtronicSerialNumberInput() {
+    const { upload } = this.props;
+    if (_.get(upload, 'source.driverId', null) !== 'Medtronic') {
+      return null;
+    }
+
+    return (
+      <div>
+        <div className={styles.textInputWrapper}>
+          <p>Enter your 6 digit serial number found on the back of your pump.</p>
+          <input
+            type="number"
```

Comment:
Ooh, good catch. Probably unlikely as you mentioned, but I can definitely add something to `onMedtronicSerialNumberInputChange`.